src/NATS.Client.Core/Internal/SslStreamConnection.cs

```diff
@@ -73,11 +96,17 @@ public void SignalDisconnected(Exception exception)
 
     public async Task AuthenticateAsClientAsync(NatsUri uri, TimeSpan timeout)
     {
+#if NET6_0_OR_GREATER
         var options = await _tlsOpts.AuthenticateAsClientOptionsAsync(uri).ConfigureAwait(true);
+#endif
         try
         {
+#if NET6_0_OR_GREATER
             using var cts = new CancellationTokenSource(timeout);
             await _sslStream.AuthenticateAsClientAsync(options, cts.Token).ConfigureAwait(false);
+#else
+            await _sslStream.AuthenticateAsClientAsync(uri.Host).ConfigureAwait(false);
```

Comment:
It looks like this effectively ignores `_tlsOpts` on `netstandard`

Reply:
I think the problem was I couldn't see na easy way of loading the PEM files

Reply:
Yea we may need to bring in Bouncy Castle or something to do it.  Not sure if someone has made a good compat library

Reply:
created https://github.com/nats-io/nats.net.v2/issues/520